nipype/interfaces/base/tests/test_traits_extension.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+from __future__ import print_function, unicode_literals
+
+from ... import base as nib
+from ..traits_extension import rebase_path_traits, resolve_path_traits, Path
+
+
+class _test_spec(nib.TraitedSpec):
+    a = nib.traits.File()
+    b = nib.traits.Tuple(nib.File(),
+                         nib.File())
+    c = nib.traits.List(nib.File())
+    d = nib.traits.Either(nib.File(), nib.traits.Float())
+    e = nib.OutputMultiObject(nib.File())
+    f = nib.traits.Dict(nib.Str, nib.File())
+    g = nib.traits.Either(nib.File, nib.Str)
+    h = nib.Str
+    ee = nib.OutputMultiObject(nib.Str)
+
+
+def test_rebase_path_traits():
+    """Check rebase_path_traits."""
+    spec = _test_spec()
+
+    a = rebase_path_traits(
+        spec.trait('a'), '/some/path/f1.txt', '/some/path')
+    assert '%s' % a == 'f1.txt'
```

Comment:
You use `Path`s elsewhere:

```suggestion
    assert a == Path('f1.txt')
```